src/pytorch_lightning/strategies/launchers/spawn.py

```diff
@@ -69,13 +78,19 @@ def launch(self, function: Callable, *args: Any, trainer: Optional["pl.Trainer"]
                 a selected set of attributes get restored in the main process after processes join.
             **kwargs: Optional keyword arguments to be passed to the given function.
         """
+        if self._start_method not in mp.get_all_start_methods():
```

Comment:
also raise this during init?

Reply:
I prefer not to. It makes the validation in accelerator connector easier, as it allows the instantiation to happen independently.